### PR TITLE
Fix #7: add gpu-elegant

### LINKED ISSUE #7: middl: install gpu-elegant
Having access to gpu-elegant could be useful for middl

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: container-conf/build.sh
Comment (robnagler) [inline container-conf/build.sh]: gpu-only (see https://github.com/radiasoft/download/pull/187/files/df755c4969a0c71814d84ca784119e0ddf58b094#r770974405)

**Review by robnagler — APPROVED** [review]
I'll let you merge after other changes are merged.

File: container-conf/build.sh
Comment (e-carlin) [reply container-conf/build.sh]: 1ebdc56

**Pull request merged**
